app/src/main/java/com/doctor/yumyum/presentation/ui/researchlist/ResearchListViewModel.kt

```diff
@@ -1,6 +1,24 @@
 package com.doctor.yumyum.presentation.ui.researchlist
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import com.doctor.yumyum.common.base.BaseViewModel
 
-class ResearchListViewModel: BaseViewModel() {
+class ResearchListViewModel : BaseViewModel() {
+    private var _sortType: MutableLiveData<Int> = MutableLiveData(1)
```

Comment:
정렬 항목을 1, 2 이렇게 하면 알아보기 어려운 것 같아서
뷰모델 안에 상수를 선언해서 사용하는건 어떤가요?

```suggestion
companion object {
    const val SORT_RECENT = 1
    const val SORT_SCRAP = 1
}
    private var _sortType: MutableLiveData<Int> = MutableLiveData(SORT_RECENT)
```
이런 식으로요..!